test/performance/odp_sched_pktio.c

```diff
@@ -718,5 +795,5 @@ int main(int argc, char *argv[])
 		return -1;
 	}
 
-	return 0;
+	return ret;
```

Comment:
I think it's better to do opposite.  C test pass if it's exit is 0. And bash should just check if it's 0 or not. That will not confuse people who run it manually. Something like return !(ret == TEST_EXPECTED_VAL).

Reply:
By default this program does not check rx/tx packet counts and returns 0. Validation test need to use special option (-s) to enable rx/tx packet statistics counting and checking. When rx and tx packet counts are checked and both are OK, return value is 3. In all other cases, return value is not 3 (but -1 or 0). E.g. if validation test script does not use -s option, we catch the error instead of returning "passed".